spec/bundler/shared_helpers_spec.rb

```diff
@@ -1,12 +1,83 @@
 require "spec_helper"
 require "bundler/shared_helpers"
 
-module TargetNamespace
-  VALID_CONSTANT = 1
-end
-
 describe Bundler::SharedHelpers do
+  describe "#default_gemfile" do
+    subject { Bundler::SharedHelpers.default_gemfile }
```

Comment:
That's valid. Don't really have any personally strong opinions on the frequency of redefining `subject`.